app/js/lib/reducers/validity.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function validity( state, action ) {
+	if ( typeof state === 'undefined' ) {
+		return {
```

Comment:
You shouldn't `return` here usually. In this case, the reducer does not work if `state` is still undefined, but you want to handle some `action`. I think that this is no problem if you use Redux correctly, since the state will be initialised beforehand anyway. It is rather the usual practice to do so.